packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/KYCNotification.js

```diff
@@ -49,16 +49,16 @@ const KYCNotification = (props) => {
       }
       <ISXContainer>
         <Text size='13px' color='brand-primary' weight={400} style={spacing('mb-20')}>
-          { header.text }
+          { state && header.text }
         </Text>
         <Text size='13px' weight={300} style={spacing('mb-20')}>
-          { body.text }
+          { state && body.text }
         </Text>
         {
-          state === 'pending' || state === 'rejected'
+          state === 'pending' || state === 'rejected' || state === 'expired'
             ? <Button onClick={() => onTrigger(kyc)} nature='empty-secondary'>
               <Text size='13px' color='brand-secondary'>
-                { kycNotificationButtonHelper(state)['text'] }
+                { state && path(['text'], kycNotificationButtonHelper(state)) }
```

Comment:
Can any of this be rendered if no kyc `state`? Maybe instead of checking everywhere individually just check `state ? <IsxContainer> : null` once?